src/Forms/ProfilesForm.php

```diff
@@ -21,10 +21,10 @@
 class ProfilesForm extends Form
 {
     /**
-     * @param string|null $entity
-     * @param array       $options
+     * @param null $entity
+     * @param array $options
      */
-    public function initialize(string $entity = null, array $options = [])
+    public function initialize($entity = null, array $options = [])
```

Comment:
@Jeckerson why remove the `string` type hint? I think it should be `?string`?

Reply:
It is mixed value and normally you pass there an model object to fill input fields inside the form.

Reply:
All good, just thought I'd ask :)